config/deployments/integration-tests/crds/vmoperator.vmware.com_virtualmachineclasses.yaml

```diff
@@ -14,7 +14,7 @@ spec:
     shortNames:
     - vmclass
     singular: virtualmachineclass
-  scope: Cluster
+  scope: Namespaced
```

Comment:
I wonder if that's the thing that breaks the integration tests.

IIRC Namespaced is the correct case, so we should adapt integration tests accordingly?

Reply:
@fabriziopandini ^^

Reply:
I'm guessing we use different CRDs for e2e tests? That would be not ideal

Reply:
🤷‍♂️ 

I don't have context on this integration tests, it is on my list to look into them and if possible get rid of them now that we have proper supervisor tests

For now, I have updated the local copy of CRDs to see if I can get them back to green

Reply:
Yup definitely a follow-up